templates/components/news-list/news-item-list.twig

```diff
@@ -26,49 +26,47 @@
 
 {% set news_url = news_url|render|striptags("<drupal-render-placeholder>")|trim %}
 
-<ul{{ attributes }}>
-  {# News List Date #}
-  {%- block block_news_list_date %}
-    {% if news_list_publishing_date|render|striptags('<drupal-render-placeholder>') is not empty %}
-      <li class="su-news-list__item-date">{{ news_list_publishing_date }}</li>
-    {% endif -%}
-  {% endblock -%}
-
+<article{{ attributes }}>
   {# News List Headline & Media #}
   {%- block block_news_list_headline  %}
-    <li class="su-news-list__item">
-      {%- if news_url is not empty -%}
-        <a {{ news_url_attributes }} href="{{ news_url }}">
-      {%- endif -%}
-      {% if news_list_image|render|striptags('<drupal-render-placeholder><img><picture>') is not empty -%}
+    <div class="news-list-header">
+    {%- if news_url is not empty -%}
+      <a {{ news_url_attributes }} href="{{ news_url }}">
+    {%- endif -%}
+        <h2>
+          {{- news_list_headline -}}
+        </h2>
+
+        {% if news_list_image|render|striptags('<drupal-render-placeholder><img><picture>')|trim is not empty -%}
         {{- news_list_image -}}
-      {%- endif -%}
-      <h2>
-        {{- news_list_headline -}}
-      </h2>
-      {%- if news_url is not empty -%}
         </a>
```

Comment:
@jenbreese ,

Will you double check your logic with the `if/endif `for this `<a>`?

It appears to me that if you don't have a url for `news_url`, this may break.

Reply:
Yup, good catch.
![image](https://user-images.githubusercontent.com/1187335/120042652-5b847c00-bfbf-11eb-8f62-b62873c0f1ff.png)

Reply:
@cjwest Fixed it and pushed it up.